Specs/DateTime/English/DateTimeModel.json

```diff
@@ -19968,5 +19968,96 @@
         }
       }
     ]
+  },
+  {
+    "Input": "I'll go back on 5.11",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "Results": [
+      {
+        "Text": "5.11",
+        "Start": 16,
+        "End": 19,
+        "TypeName": "datetimeV2.date",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "XXXX-05-11",
+              "type": "date",
+              "value": "2016-05-11"
+            },
+            {
+              "timex": "XXXX-05-11",
+              "type": "date",
+              "value": "2017-05-11"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "I'll go back on 5.1935",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "Results": []
+  },
+  {
+    "Input": "I'll go back 5.11",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "Results": []
+  },
+  {
+    "Input": "I'll go back during 5.11",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "Results": []
+  },
+  {
+    "Input": "I'll go back at 5.11",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "Results": [
+      {
+        "Text": "at 5.11",
+        "Start": 13,
+        "End": 19,
+        "TypeName": "datetimeV2.time",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "T05:11",
+              "type": "time",
+              "value": "05:11:00"
+            },
+            {
+              "timex": "T17:11",
+              "type": "time",
+              "value": "17:11:00"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "I'll go back at 5.85",
```

Comment:
Please reword this to "The price ended the day at 5.85".

Reply:
done